### ISSUE TITLE
Notifications: Find place for new dedictated notifications screen

### ISSUE THREAD

**Issue by eoigal** [Automattic/wp-calypso#71131]
### Details

We need to find a place for the new dedicated notifications screen.

I think we should redesign https://wordpress.com/me/notifications completely and use this page for the users notifications.
We will need to move the notification settings (that currently resides at https://wordpress.com/me/notifications).

<img width="1418" alt="Screenshot 2022-12-13 at 12 38 57" src="https://user-images.githubusercontent.com/5560595/207320593-64f722af-c441-4626-ac95-37b5ddefa2ee.png">

One issue I see is we currently appear to combine notification settings in one place.
Notification settings are a catch-all (or dumping ground) for all types of notifications settings and I think it could do with a re-organisation.

A user can disable a bell/web notification, an email or a mobile push notification for activity on their blogs or when they comment on a blog.

<img width="1428" alt="Screenshot 2022-12-13 at 12 39 12" src="https://user-images.githubusercontent.com/5560595/207321058-955921a1-78f8-4d8b-8d73-503d9c7dab61.png">

They can also opt out of emails from WPCOM and Jetpack.

<img width="1426" alt="Screenshot 2022-12-13 at 12 39 25" src="https://user-images.githubusercontent.com/5560595/207321110-ff0c8d8f-eab8-4b25-8ee8-b9fb190afef2.png">

They can also set default notification behaviour for reader subscription notifications.

<img width="1422" alt="Screenshot 2022-12-13 at 12 39 47" src="https://user-images.githubusercontent.com/5560595/207321137-0d0bf1d0-0a49-4888-8297-71d647f7a75a.png">

If we agree to use https://wordpress.com/me/notifications as the dedication notifications screen, what impact will it have to move the settings to a new URL?

### Checklist

- [ ] Agree on location for dedicated notifications screen

**Comment by ollierozdarz** [comment]
@eoigal there's been an update to the original design P2.

TL;DR:
> This feature will be located in https://wordpress.com/me under ../me/notifications. We’ll change the navigation label from Notification Settings to Notifications, and include a sub-link to the Settings page below it.

![image](https://user-images.githubusercontent.com/67085494/207718822-ef9d6e24-6cf6-4f83-878e-2b1c2cc4e719.png)
